### DISCUSSION TITLE
round single corner xaml

### DISCUSSION THREAD

**Discussion by bebiksior** [xamarin/Xamarin.Forms#12445] [Q&A]
hi, how can I round single corner

tnx

**Comment by jsuarezruiz** [comment]
For example, using `BoxView` you can adjust each corner.

**Comment by GalaxiaGuy** [comment]
In general it is not supported. There is `ICornerElement` which has `CornerRadius` which let you specify different values for each corner, but only `BoxView` supports it.

There is the additional issue that `Button` has `int` values and `Frame` has `float` values which can cause problems.

It would be nice to use `CornerRadius` everywhere, but I don't think it can be done without being a breaking change. Hopefully this could be considered for Maui.

**Comment by Cfun1** [comment]
Try using [Xamarin.Forms.PancakeView](https://github.com/sthewissen/Xamarin.Forms.PancakeView) package, example in [Stackoverflow: Only rounded bottom corners?](https://stackoverflow.com/a/64742108/5228202)
